src/vcBPA.cpp

```diff
@@ -78,13 +78,12 @@ struct vcBPAGrid
     return grid;
   }
 
-  void Init(bool initPointBuffer = true)
+  void Init(vdkStandardAttributeContent standardContent)
```

Comment:
I'll have a chat with you offline but I think this should be a `vdkStandardAttributeSet`

Reply:
After offline chat, this is correct for the current version of VDK0.5.0